docs/examples/video/video_label_example.py

```diff
@@ -41,6 +41,8 @@ class VideoPipe(Pipeline):
     def __init__(self, batch_size, num_threads, device_id, data):
         super(VideoPipe, self).__init__(batch_size, num_threads, device_id, seed=12)
         self.input = ops.VideoReader(device="gpu", file_root = data, sequence_length=COUNT,
+        # instead of file_root, path to text file with pairs video_filepath label_value can be provided
+        # self.input = ops.VideoReader(device="gpu", file_list = "file_list.txt", sequence_length=COUNT,
```

Comment:
Maybe remove it? This info can be found in docs and here it disrupts a flow a bit, I think.

Reply:
We got this for the file reader, and it gives the easy way to just switch one option to the other without wondering how to do it.